build.gradle

```diff
@@ -11,6 +11,7 @@ targetCompatibility = JavaVersion.VERSION_17
 repositories {
     mavenLocal()
     maven { name="BuildCraft"; url="https://mod-buildcraft.com/maven" }
+    maven { url="https://maven.pkom.ml/" }
```

Comment:
This is going to need to be changed to point to the official maven, but I can do that once LNS and LBA are published.